criu/net.c

```diff
@@ -2150,9 +2150,8 @@ static int dump_netns_conf(struct ns_id *ns, struct cr_imgset *fds)
 		i++;
 
 	o_buf = buf = xmalloc(i * (sizeof(NetnsId *) + sizeof(NetnsId)) +
-			      size4 * (sizeof(SysctlEntry *) + sizeof(SysctlEntry)) * 2 +
-			      size6 * (sizeof(SysctlEntry *) + sizeof(SysctlEntry)) * 2 +
-			      sizex * (sizeof(SysctlEntry *) + sizeof(SysctlEntry)));
+			      (size4 * 2 + size6 * 2 + sizex) *
+			      (sizeof(SysctlEntry *) + sizeof(SysctlEntry)));
```

Comment:
@Snorch Would it be possible to add a comment explaining these lines?

Reply:
Done.